src/components/chat/commented-documents.tsx

```diff
@@ -96,7 +84,19 @@ export const CommentedDocuments: React.FC<IProps> = ({user, handleDocView}) => {
     return () => unsubscribeFromDocs?.();
   },[cDocsRef, cDocsInScopeRef]);
 
-  // ------MyWork/ClassWork--------
+
+  //------Documents: (i.e. //"Student Workspaces/"My Work"/"Class Work")
```

Comment:
Moved code around so that Curriculum Documents (problem/teacher guide)  `useState` and `useEffect` were above Document Documents (student workspace/my work/ class work") `useState` and `useEffect`